### DISCUSSION TITLE
Dbeaver export help

### DISCUSSION THREAD

**Discussion by michoglen63** [dbeaver/dbeaver#10822] [Q&A] [answered]
Hello,

I am curious to know if the is a setting in Dbeaver to when I export the data, the query used will show on a second worksheet in excel.

I know it works in sql developer but was unsure if the option was available in Dbeaver?

Thanks https://100001.onl/

**Comment by uslss** [comment, marked as answer]
Hi @michoglen63 ,
Unfortunately, it is not implemented yet, but you can create a feature request.
Best regards
